server/server.go

```diff
@@ -103,14 +138,17 @@ func (s *Server) HandleReview(ctx context.Context, e *ReviewEvent) error {
 	}
 	comments := s.concurrentRequest(ctx, logger, conf, send)
 
-	s.post(ctx, logger, e, comments)
-	s.status(ctx, logger, e, SuccessAnalysisStatus)
+	if err := s.post(ctx, logger, e, comments); err != nil {
+		return fmt.Errorf("posting analysis failed: %s", err)
```

Comment:
I think this will leave the github status in pending, and we should set it to lookout.ErrorAnalysisStatus.

Reply:
true. Thanks!